src/globalComponents/apiCalls/getUserAPICall.js

```diff
@@ -2,27 +2,28 @@
 
 import getLoginToken from '../getLoginToken.js';
 import User from '../typesAndClasses/User.js';
+import { getInventory } from '../GetInventoryInfo.js';
 
 //gets the user when passed a token stored as the login token
 function getUserAPICall(onLoad:(user: User) => void): void {
-	const token=getLoginToken();
 	const responsePromise: Promise<Response> = fetch('/api/user/getUser', {
 		method: 'GET',
 		headers: {
 			'Access-Control-Allow-Origin': '*',
 			'Content-Type': 'application/json',
 			'Access-Control-Allow-Credentials': 'true',
-			'x-auth-token': token
+			'x-auth-token': getLoginToken()
 		},
 	});
 	responsePromise.then (
 		response => response.json().then(data => {
 			if (response.status !== 200) {
 				console.log(response.status);
 				console.log(data);
+				onLoad(data);
```

Comment:
hmm, this seems sketchy. You aren't actually returning a user here. You shouldn't call onLoad, and you should probably just do toast.error() here.

Reply:
Sounds good.